runtime-api/src/main/scala/io/coral/actors/Messages.scala

```diff
@@ -4,26 +4,34 @@ import akka.actor.{ActorRef, ActorPath}
 import org.json4s._
 
 object Messages {
-  // Generic messages
-
-
-  // RuntimeActor messages
-  case class CreateActor(json: JObject)
-  case class RegisterActorPath(id: Long, path: ActorPath)
-  case class GetCount()
-  case class ListActors()
-  case class GetActorPath(id: Long)
-  case class Delete(id: Long)
-  case class DeleteAllActors()
-
-  // CoralActor messages
-  case class Get()
-  case class Request(json: JObject)
-  case class GetField(field:String)
-  case class ListFields()
-  case class RegisterActor(r: ActorRef)
-  case class UpdateProperties(json:JObject)
-  case class GetProperties()
-
-  case object TimeoutEvent
+    // RuntimeActor messages
+    case class CreateActor(json: JObject)
+    case class RegisterActorPath(id: Long, path: ActorPath)
+    case class GetCount()
+    case class ListActors()
+    case class GetActorPath(id: Long)
+    case class Delete(id: Long)
+    case class DeleteAllActors()
+
+    // CoralActor messages
```

Comment:
The use case is testing. It is useful to execute the emit function specifically, and nothing else, from a unit test. This way, it is very easy to intercept and check an emit result.